commands/api.go

```diff
@@ -290,7 +293,26 @@ func apiCommand(cmd *Command, args *Args) {
 			}
 		}
 		if requestLoop && !parseJSON {
-			fmt.Fprintf(out, "\n")
+			_, _ = fmt.Fprintf(out, "\n")
```

Comment:
What does assigning to `_, _` accomplish?

Reply:
It was part of the overall "explicit error handling" that made GoLand happy. That's where the rest of those error handling things came from, too: GoLand considers it ... suspicious ... if a call returns `error` but no one checks it